core/templates/dev/head/pages/library/library.html

```diff
@@ -43,7 +43,39 @@
           <search-results></search-results>
         </div>
 
-        <div class="oppia-exp-summary-tiles-container" ng-if="!inSearchMode">
+        <div class="oppia-exp-summary-tiles-container" ng-if="inGroupMode" style="margin-bottom: 20px;">
+          <div ng-if="libraryGroups.length == 0"
+               style="font-size: 1.2em; padding-bottom: 40px; padding-top: 40px;">
+              <p style="text-align: center;">
+                <span>Oops, there are no explorations to display.</span>
```

Comment:
How do I translate this string in other languages? I do not know it's translation in other languages.